shared/altair-devnet-0/bootnode.txt

```diff
@@ -0,0 +1,3 @@
+# EF bootnode
```

Comment:
I think this should be a YAML file called `boot_enr.yaml` (see `teku/yeerongpilly/boot_enr.yaml`).

Reply:
On mainnet this is a text file - yaml would be something new? https://github.com/eth2-clients/eth2-networks/blob/master/shared/mainnet/bootstrap_nodes.txt

Reply:
the name is off though, should be `bootstrap_nodes.txt`

Reply:
ah yeah, Paul pointed out that it's just Lighthouse and Teku using that YAML format, sorry

Reply:
not that I mind, but we submodule the repo and load the files at compile time with a magic process that relies on names and formats - much easier if it's consistent across all network - ie a migration to yaml would certainly be possible but then it should cover all networks ideally in one big happy commit

Reply:
I've renamed it to `bootstrap_nodes.txt` for now so it's in line with the mainnet format. Would you prefer that in the future I replicate the content once in a `yaml` file and once in a `txt` file or do we leave it in `txt` until all clients have migrated to a common format?

Reply:
I don't think having two files is a good idea - we already had issues with stale boot nodes because of a similar setup - I'd assume `txt` files are the common format for now, unless there was a discussion somewhere that I missed?

Reply:
`txt` is fine, for now we'll just keep duplicating as YAML

Reply:
alright, I will stick to `txt` as well